nunit.sln

```diff
@@ -1,8 +1,6 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.23107.0
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 2012
```

Comment:
Personally, I really like 2015 and the Community Edition is a great
offering. Besides, if we are going to support newer platforms, you are
going to have to move ;-)

On 20 October 2015 at 13:55, CharliePoole <notifications@github.com> wrote:

> In nunit.sln
> <https://github.com/nunit/nunit/pull/909#discussion_r42529966>:
>
> > @@ -1,8 +1,6 @@
> >  ﻿
> >  Microsoft Visual Studio Solution File, Format Version 12.00
> > -# Visual Studio 14
> > -VisualStudioVersion = 14.0.23107.0
> > -MinimumVisualStudioVersion = 10.0.40219.1
> > +# Visual Studio 2012
>
> With a bit more seriousness: I have always tried to keep using the lowest
> version of the IDE that will work for building NUnit. That keeps it working
> for others who may want to contribute but can't afford Microsoft prices.
> The VS2015 community edition may change my approach though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/909/files#r42529966>.
>



-- 

Rob Prouse



I welcome VSRE emails. Learn more at http://vsre.info/